skaffold/skaffold_cloud.yaml

```diff
@@ -153,3 +153,9 @@ profiles:
     path: /manifests/kustomize/paths
     value:
     - k8s/cloud/public/base
+- name: x86_64_sysroot
+  patches:
+  - op: add
+    path: /build/artifacts/context=./bazel/args
+    value:
+    - --config=x86_64_sysroot
```

Comment:
I this this doesn't stack so if you do prod and sysroot, you'll lose opt and stamping etc.
You might want to add this to the prod profile

Reply:
Good call. I glossed over that. I've updated it for testing, staging and prod and used a yaml anchor to reduce the duplication.